.gitignore

```diff
@@ -5,6 +5,8 @@ venv/
 __pycache__/
 *.py[cod]
 
+/client/.eslintrc
```

Comment:
Why are you gitignoring this?

Reply:
not sure why it came up in my repo as an untracked file. thought something autogenerated it